static/src/javascripts/projects/common/modules/commercial/first-pv-consent-plus-fiv-banner.js

```diff
@@ -0,0 +1,136 @@
+// @flow
+import { Message } from 'common/modules/ui/message';
+import type { Banner } from 'common/modules/ui/bannerPicker';
+import {
+    messageCode as fivMessageCode,
+    canShow as canShowFivBanner,
+    defaultEngagementBannerParams,
+    getBannerHtml as getFivBannerHtml,
+} from 'common/modules/commercial/fiv-banner';
+import {
+    track as trackFirstPvConsent,
+    bindClickHandlers as bindFirstPvConsentClickHandlers,
+    canShow as canShowFirstPvConsent,
+    messageCode as firstPvConsentMessageCode,
+    makeHtml as makeFirstPvConsentHtml,
+} from 'common/modules/ui/first-pv-consent-banner';
+import marque36icon from 'svgs/icon/marque-36.svg';
+
+const messageCode: string = 'first-pv-consent-plus-fiv-banner';
+
+const bannerParams: EngagementBannerParams = defaultEngagementBannerParams();
+
+const getBannerHtml = (acquisitionsBannerFivHtml: string) => `
+    <div class="site-message js-site-message js-double-site-message site-message--banner site-message--double-banner" tabindex="-1" role="dialog" aria-label="welcome" aria-describedby="site-message__message" data-component="AcquisitionsEngagementBannerStylingTweaks_control">
+        <div class="js-fiv-banner-site-message site-message--engagement-banner site-message--fiv-banner">
+            <div class="gs-container">
+                <div class="site-message__inner js-site-message-inner">
+                    <div class="site-message__roundel">
+                        ${marque36icon.markup}
+                    </div>
+                    <div class="site-message__copy js-site-message-copy u-cf">
+                        ${acquisitionsBannerFivHtml}
+                    </div>
+                </div>
+            </div>
+        </div>
+    
+        <div class="js-first-pv-consent-site-message site-message--first-pv-consent" tabindex="-1" data-link-name="release message" role="dialog" aria-label="welcome" aria-describedby="site-message__message">
+            <div class="gs-container">
+                <div class="site-message__inner js-site-message-inner">
+                    <div class="site-message__roundel">
+                        ${marque36icon.markup}
+                    </div>
+                    <div class="site-message__copy js-site-message-copy u-cf">
+                        ${makeFirstPvConsentHtml()}
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+`;
+
+class SubMessage extends Message {
```

Comment:
this file heavily copies from `first-pv-consent-plus-engagement-banner.js`
In future if we need to do this more then we should do something more generic